products/community-edition.md

```diff
@@ -2,40 +2,40 @@
+There are several opportunities to run Shopware in general. When speaking of the Community Edition, let's be more precise about the **Shopware Platform**, which can be included inside several [setup templates](../guides/installation/overview.md#setup-templates) or can be shipped as a One-Click-Installer. Of course, there are other operation forms like Docker, Vagrant, Valet+, which are explained in the [installation guide](../guides/installation/) section. Inside that section, you will also find the [system requirements](../guides/installation/overview.md#prerequisites) needed for running Shopware as a PHP application built upon Symfony.
 
 ## Platform components
 
-The Shopware Platform itself is a Symfony application which consists of several components developed as Symfony bundles. Each of these components is available throughout a many repository and is included in the Shopware Platform mono repository. For the time being, there is the Core component, which includes the framework and business logic, as well as the APIs. The [Storefront](../guides/plugins/plugins/storefront/) component is a default frontend for your storefront built upon the Bootstrap toolkit and Twig templates. A Vue.js SPA [Administration](../concepts/framework/architecture/administration-concept.md) component wrapped inside a Symfony bundle is a default administration panel for all back-office tasks and communicates via the [Admin API](../concepts/api) with the Core component. Last but not least, there is the Elasticsearch component, which gives you the opportunity to improve indexing of entities and also contains an adapter for the entity search.
+The Shopware Platform itself is a Symfony application which consists of several components developed as Symfony bundles. Each of these components is available throughout many repositories and is included in the Shopware Platform mono repository. For the time being, there is the Core component, which includes the framework and business logic, as well as the APIs. The [Storefront](../guides/plugins/plugins/storefront/) component is a default frontend for your Storefront built upon the Bootstrap toolkit and Twig templates. A Vue.js SPA [Administration](../concepts/framework/architecture/administration-concept.md) component wrapped inside a Symfony bundle is a default Administration panel for all back-office tasks and communicates via the [Admin API](../concepts/api) with the Core component. Last but not least, the Elasticsearch component gives you the opportunity to improve the indexing of entities and also contains an adapter for the entity search.
```

Comment:
```suggestion
The Shopware Platform itself is a Symfony application which consists of several components developed as Symfony bundles. Each of these components is mirrored to a many repository and included in the Shopware Platform mono repository. For the time being, there is the Core component, which includes the framework and business logic, as well as the APIs. The [Storefront](../guides/plugins/plugins/storefront/) component is a default frontend for your Storefront built upon the Bootstrap toolkit and Twig templates. A Vue.js SPA [Administration](../concepts/framework/architecture/administration-concept.md) component wrapped inside a Symfony bundle is a default Administration panel for all back-office tasks and communicates via the [Admin API](../concepts/api) with the Core component. Last but not least, the Elasticsearch component gives you the opportunity to improve the indexing of entities and also contains an adapter for the entity search.
```

Reply:
It's actually called a "many/multi repository" and a "mono repository" - it's a pattern of organizing bigger software projects. 

https://en.wikipedia.org/wiki/Monorepo

Reply:
Okay :)